{"name": "test_add[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert -0.1 == 0.3", "trace": "self = <pytest_practice.testcases.test_calc.TestCalcutor object at 0x03E64A78>\nget_calc = <pytest_practice.common.calc.Calculator object at 0x03E64880>\nget_add_datas = [0.1, -0.2, 0.3]\n\n    @allure.story(\"测试加法\")\n    def test_add(self, get_calc, get_add_datas):\n        # 步骤说明\n        with allure.step(\"计算两个数的和\"):\n            result = get_calc.add(get_add_datas[0],get_add_datas[1])\n        if isinstance(result, float):\n            result = round(result, 2)\n>       assert result == get_add_datas[2]\nE       assert -0.1 == 0.3\n\ntestcases\\test_calc.py:20: AssertionError"}, "steps": [{"name": "计算两个数的和", "status": "passed", "start": 1612096144402, "stop": 1612096144402}], "parameters": [{"name": "get_add_datas", "value": "[0.1, -0.2, 0.3]"}], "start": 1612096144402, "stop": 1612096144402, "uuid": "a7bd695d-611c-41cd-b480-4dd0089e3976", "historyId": "feb623152a4aaa4876d754c159f3a51e", "testCaseId": "f57a9b88ab5525b904d9823ce49f6858", "fullName": "testcases.test_calc.TestCalcutor#test_add", "labels": [{"name": "story", "value": "测试加法"}, {"name": "feature", "value": "计算器测试"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalcutor"}, {"name": "host", "value": "XTZ-01705042203"}, {"name": "thread", "value": "12664-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_calc"}]}